sensor_msgs/JointState joints
---
bool isColliding